varats/varats/experiments/vara/feature_experiment.py

```diff
@@ -31,6 +40,23 @@
 from varats.report.report import ReportSpecification
 
 
+class FeatureInstrType(Enum):
+    """Type of instrumentation to be used in feature tracing ."""
+
+    NONE = ""
+    """Don't add any instrumentation."""
+    PRINT = "print"
+    """Print tracing information to stdout."""
+    TEF = "trace_event"
+    """Produce a trace file."""
+    USDT = "usdt"
+    """Insert USDT probes."""
+    USDT_RAW = "usdt_raw"
+    """Insert raw USDT probes."""
+    VERIFY = "instr_verify"
+    """Verify markers."""
```

Comment:
```suggestion
    """Don't add any instrumentation."""
    NONE = ""
    """Print tracing information to stdout."""
    PRINT = "print"
    """Produce a trace file."""
    TEF = "trace_event"
    """Insert USDT probes."""
    USDT = "usdt"
    """Insert raw USDT probes."""
    USDT_RAW = "usdt_raw"
    """Verify markers."""
    VERIFY = "instr_verify"
```
we normally put comments above

Reply:
These are intended as doc strings that should be visible from IntelliSense. If you put them above then at least VS Code doesn't correctly relate them.

Reply:
That's weird, if find placing string "after" the value strings below the var as highly confusing :thinking:
1) I read the name/value and afterwards can only see the doc -> reading flow is bad
2) If the string below somehow get's formatted to the above (or the line ends in `\`) the values change. But this would, without new lines, also be the case for the above variant.

However, if this is the only option to get the docs to work, we should at least add linebreaks for the comments and an empty line inbetween.

```
class Foo(Enum):

    FOO = "f"
    """
    Foo doc
    """

    BAR = "b"
    """
    Bar doc
    """
```

Reply:
Unfortunately, they indeed have to be after the variable when using docstrings.
However, there is an alternative with special comments we could try.
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute

Reply:
I quickly tried replacing the doc strings with comments starting with `#:  ` as @boehmseb suggested. The issue with this approach is that VS Code's Intellisense feature doesn't handle them resulting in no documentation at all being shown. In my opinion, if you don't have another idea, the best option is to stick with doc strings below the declarations. I already inserted the newlines.